src/valkey/valkey.zig

```diff
@@ -5,9 +5,13 @@
 pub const ValkeyContext = @import("./ValkeyContext.zig");
 
 /// Connection flags to track Valkey client state
-pub const ConnectionFlags = packed struct(u8) {
+pub const ConnectionFlags = packed struct(u9) {
+    // TODO(markovejnovic): I am not a huge fan of these flags. I would
```

Comment:
agree, and it also doesn't need to be a packed struct because we aren't using it anywhere that padding wouldn't fill in the remaining space.